blueprints/README.md

```diff
@@ -33,7 +33,7 @@ $ linuxkit build -name docker-for-mac base.yml docker-ce.yml
 To run the VM with a 4G disk:
 
 ```
-linuxkit run hyperkit -networking=vpnkit -vsock-ports=2376 -disk size=4096M -data ./metadata.json docker-for-mac
+linuxkit run hyperkit -networking=vpnkit -vsock-ports=2376 -disk size=4096M -data-path ./metadata.json docker-for-mac
```

Comment:
I think I would prefer `-data-file` as it seems clearer that this is for a single file.

Reply:
I am ok with that. No big deal to change.

Reply:
Done `data-file`